.github/E2E_TESTS_ON_GITHUB_CI.md

```diff
@@ -1,58 +1,87 @@
 # E2E Tests on GitHub CI
 
-E2E tests are executed on GitHub CI using the [E2E Tests Reusable Workflow](#about-the-reusable-workflow) or dedicated workflows.
+- [E2E Tests on GitHub CI](#e2e-tests-on-github-ci)
+  - [Scheduled test workflows](#scheduled-test-workflows)
+    - [PR E2E Tests](#pr-e2e-tests)
+    - [Nightly E2E Tests](#nightly-e2e-tests)
+    - [Release E2E Tests](#release-e2e-tests)
+      - [Smoke Tests](#smoke-tests)
+      - [Client Compatibility Tests](#client-compatibility-tests)
+  - [On-Demand Workflows](#on-demand-workflows)
+    - [Test workflows setup in CI](#test-workflows-setup-in-ci)
+      - [Configuration Overrides](#configuration-overrides)
+      - [Test Secrets](#test-secrets)
 
-## Automatic workflows
+E2E tests are executed on GitHub CI using the [E2E Tests Reusable Workflow](https://github.com/smartcontractkit/.github/blob/main/.github/workflows/README.md) or dedicated workflows.
 
-These workflows are designed to run automatically at crucial stages of the software development process, such as on every commit in a PR, nightly or before release.
+## Scheduled test workflows
+
+These workflows are designed to run on every commit in a PR, nightly or before release (see `triggers` in [e2e-tests.yaml](./e2e-tests.yml)).
 
 ### PR E2E Tests
 
-Run on every commit in a PR to ensure changes do not introduce regressions.
+Smoke tests triggered on every commit in a PR to ensure changes do not introduce regressions.
```

Comment:
@chudilka1 these are not only smoke tests. In fact, we run almost all tests we have except soak or performance tests.

Reply:
@lukaszcl looking at https://github.com/smartcontractkit/chainlink/blob/develop/.github/e2e-tests.yml every single test that is in either `PR E2E Core Tests` or `Merge Queue E2E Core Tests` are all smoke tests. The only test that is not a smoke is `migration/upgrade_version_test.go`, but even that is more or less like a smoke, but just checks upgrade.

Reply:
Updated, removed - Smoke